content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance"]
+---
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog FIPS proxy is only available in the US1-FED region.</a></div>
+{{< /site-region >}}
+
+The Datadog FIPS proxy ensures that communication between the Datadog Agent and Datadog uses FIPS-compliant encryption.
```

Comment:
If we're considering the FIPS proxy as a product, it may be more appropriate to call it universally "Datadog Agent FIPS Proxy" including capitalization (or something similar) but that's more of a subjective call